src/fixate/drivers/dso/helper.py

```diff
@@ -12,7 +12,7 @@
     number = float
 
 
-def open(restrictions=None):
+def open():
```

Comment:
All the driver/*/helper.py files reference discover_sub_classes. I'm not sure if the discover functions get used anywhere, but I think discover_sub_classes is broken.

discover_sub_classes references fixate.config.CLASS_LIST. As far as I can tell, it only ever gets initialised to None. I'm 100% though, since there might be some non-obvious reference. But possibly another item that can be cleaned up?

Reply:
Yeah it can be cleaned up. DRIVER_LIST was added to explicitly define the implemented classes which made the discover classes unused.